rust/numaflow-core/src/pipeline.rs

```diff
@@ -267,11 +270,10 @@ async fn create_transformer(
         if let config::components::transformer::TransformerType::UserDefined(ud_transformer) =
             &transformer_config.transformer_type
         {
-            check_for_server_compatibility(
-                ud_transformer.socket_path.clone().into(),
-                cln_token.clone(),
-            )
-            .await?;
+            _ = sdk_server_info(ud_transformer.socket_path.clone().into(), cln_token.clone())
+                .await?;
```

Comment:
```suggestion
            _ = sdk_server_info(ud_transformer.socket_path.clone().into(), cln_token.clone())
                .await?;
```

do we need this change?

Reply:
Do we need to handle the the return in rust? Also, I assume something needs to be added to for the returned server info later on - this is for pipeline.

Reply:
`?` does the return on error. the success is a zero-size-type `()` which has no meaning.

Reply:
The success is changed to ServerInfo.

Reply:
then it is fine, i missed that